src/OrchardCore.Modules/OrchardCore.Localization/Manifest.cs

```diff
@@ -22,3 +22,11 @@
     Dependencies = new[] { "OrchardCore.Localization" },
     Category = "Internationalization"
 )]
+
+[assembly: Feature(
+    Id = "OrchardCore.Localization.CulturePicker.Admin",
+    Name = "Admin Culture Picker",
+    Description = "Provides a culture picker shape for the backend.",
```

Comment:
```suggestion
    Description = "Provides a culture picker shape for the admin area.",
```

Reply:
I used backed, similar to content localization that using frontend, but it's fine

Reply:
I used backed, similar to content localization that using frontend, but it's fine